source/elements/oneDAL/source/data_management/accessors.rst

```diff
@@ -1,3 +1,85 @@
+.. _accessors:
+
 =========
 Accessors
 =========
+
+This section contain definitions of classes that implement :ref:`accessor`
+concept.
+
+------------------------------
+Requirements on accessor types
+------------------------------
+
+Each implementation of :ref:`accessor` concept in |dal_short_name| shall follow
+next requirements:
+
+1. Define single :term:`format of the data <Data format>` of the
+   accessor. Every single accessor type shall provide only one :term:`format of
+   the data <Data format>` returned from it or put inside it.
+
+2. Allow to make access to at least of one in-memory :ref:`dataset`
+   implementation (like :code:`table` or its sub-types or :ref:`table builders
+   <table-builder>`).
+
+3. Allow to make read-only, write-only and read-write access to the data. If
+   accessor supports several :ref:`dataset` implementations to be passed in, it
+   is not necessary to support all access modes for every input object. For
+   example, tables shall support single read-only mode according to their
+   :ref:`concept <table>` definition.
+
+4. Shall name reading and writing operations consistently. Reading operation
```

Comment:
provide the names for read and write operations following the pattern:
- pull_  for reading
- push_ for..